index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>SINGOLO*</title>
+      <link rel="stylesheet" href="css\resetEricMayer.css">
+      <link rel="stylesheet" href="css\style.css">
+</head>
+<body>
+      <div class="shadow"></div>
+      <header>
+            <section>
+                  <article class="header-article">
+                        <h1>SINGOLO<span>*</span></h1>
+                        <nav>
+                              <a href="#home" class="nav-buttons nav-buttons_active header__nav-buttons">HOME</a>
+                              <div class="nav__separator"></div>
+                              <a href="#services" class="nav-buttons header__nav-buttons">SERVICES</a>
+                              <div class="nav__separator"></div>
+                              <a href="#portfolio" class="nav-buttons header__nav-buttons">PORTFOLIO</a>
+                              <!-- <div class="nav__separator"></div>
+                              <a href="#~" class="nav-buttons disabled">ABOUT</a>
+                              <div class="nav__separator"></div>
+                              <a href="#~" class="nav-buttons disabled">CONTACT</a> -->
+                        </nav>
+                  </article>
+            </section>
+      </header>
+      <div class="burger burger_active">
+            <div class="burger__head">
+                  <div class="burger__icon">
+                        <span></span>
+                  </div>
+                  <h2>SINGOLO<span>*</span></h2>
+            </div>
+            
+            <nav>
+                  <a href="#home" class="nav-buttons nav-buttons_active">HOME</a>
+                  <a href="#services" class="nav-buttons">SERVICES</a>
+                  <a href="#portfolio" class="nav-buttons">PORTFOLIO</a>
+                  <!-- <a href="#~" class="nav-buttons disabled">ABOUT</a>
+                  <a href="#~" class="nav-buttons disabled">CONTACT</a> -->
+            </nav>
+      </div>
+
+      <main>
+            <section class="main-slider" id="home">
+                  <article class="main-slider__article">
+                      <div class="slider">
+                          <div class="slider__wrapper">
+                            <div class="slider__items">
+                              <div class="slider__item">
+                                  <img src="images/Slider1.png" alt="slide-1">

```

Comment:
alt=slide-1 don't have value if the image doesn't load.

Reply:
fixed